### PR TITLE
add hooks-by-example

### PR DESCRIPTION
**Pull request by latviancoder**
Hey, I wasn't sure which section to put this into. I hope it's okay.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,7 @@
 - [Hooks.guide](https://www.hooks.guide/) Collection of React hooks curated by the community.
 - [Searchable Collection of React Hooks](https://nikgraf.github.io/react-hooks/)
 - [useHooks(🐠)](https://usehooks.com/) One new React Hook recipe every day.
+- [hooks-by-example](https://github.com/latviancoder/hooks-by-example) Collection of beginner-friendly real world examples of hook usage.
```
Comment (aarongarciah) [inline README.md]: Please add it in the first place in order to keep the links sorted alphabetically :)

File: README.md [hunk 2]
```diff
@@ -47,7 +48,7 @@
 - [`@rehooks/online-status`](https://github.com/rehooks/online-status) React Hook for Online status.
 - [`@rehooks/window-scroll-position`](https://github.com/rehooks/window-scroll-position) React hook for getting window `x` and `y` position.
 - [`@rehooks/window-size`](https://github.com/rehooks/window-size) React hook for subscribing to window size.
-- [`easy-peasy`](https://github.com/ctrlplusb/easy-peasy) Easy peasy global state for React.
+- [`easy-zpeasy`](https://github.com/ctrlplusb/easy-peasy) Easy peasy global state for React.
```
[the remark this answers is not in the document]
Comment (latviancoder) [reply README.md]: Fixed :)

File: README.md (hunk 1 above)
Comment (latviancoder) [reply README.md]: Done

**Comment by aarongarciah** [comment]
Thanks @latviancoder  😄

**Pull request merged**